rmw_implementation/package.xml

```diff
@@ -19,11 +21,11 @@
   <build_depend>rmw_connext_cpp</build_depend>
   <build_depend>rmw_cyclonedds_cpp</build_depend>
   <build_depend>rmw_fastrtps_cpp</build_depend>
+  <build_depend>rmw_implementation_cmake</build_depend>
   <!-- end of group dependencies added for bloom -->
 
   <depend>libpoco-dev</depend>
   <depend>poco_vendor</depend>
-  <depend>rmw_implementation_cmake</depend>
```

Comment:
This depenency is being used by downstream packages, see https://github.com/ros2/rmw_implementation/blob/86dd12ae741fa990c3e16ce102915f2d80b8013a/rmw_implementation/rmw_implementation-extras.cmake.in#L17

So removing this is breaking the build, e.g.: https://ci.ros2.org/job/ci_linux/10045/